service/controller/v23/resource/bridgezone/resource.go

```diff
@@ -243,7 +243,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 		upsert := route53.ChangeActionUpsert
 		ns := route53.RRTypeNs
-		ttl := int64(900)
+		ttl := int64(300)
```

Comment:
For the record, I lowered this to improve e2e reliability. The AWS docs say that lower TTLs come with cost increases but I anticipate this is negligible.

Reply:
I think it should go to a separate PR.

Reply:
I agree. Do you want me to provide a separate PR for this? Otherwise I would merge it like it is.